Atomic/mount.py

```diff
@@ -732,6 +732,28 @@ def _clean_temp_container_by_path(self, path):
             self.d.remove_container(short_cid)
         self._clean_tmp_image()
 
+
+class DockerMountContextManager(object):
```

Comment:
These are great suggestions! Maybe I could move that code into separate PR. It will likely take me a couple days.